.github/workflows/build_test_push.yaml

```diff
@@ -64,12 +66,13 @@ jobs:
             ${{ runner.os }}-graphhopper-buildx
       # ---------------------
 
-        # Build image for unit tests
-      - name: Build test image
+      # Build base image for unit tests
+      - name: Build base image
         uses: docker/build-push-action@v2
         with:
           context: .
-          push: true
+          push: false
+          load: true
```

Comment:
`load: true` is what tells docker to push/pull from the local docker repo. vs remote